draft-ietf-rats-architecture.md

```diff
@@ -844,40 +835,89 @@ a different format.
 
 # Freshness {#freshness}
 
-It is important to prevent replay attacks where an attacker replays
-old Evidence or an old Attestation Result that is no longer correct.
-To do so, some mechanism of ensuring that the Evidence and Attestation
-Result are fresh, meaning that there is some degree of assurance
-that they still reflect the latest state of the Attester, and that any Attestation
-Result was generated using the latest Appraisal Policy for Evidence.
+It is important to prevent old Evidence or old Attestation Results that are no longer correct to be assessed as valid. Replay attacks are an important reason to do that. More importantly, the validity of Conceptual Messages, such as Evidence or Attestation Results, are the basis for trustworthy remote attestation procedures, in general.
+Some mechanism of ensuring that Evidence and Attestation
+Results are actually fresh are required. In essence, there has to be some degree of assurance
+that they still reflect the latest operational state of the Attester and that, for example, Attestation
+Results were generated using the latest Appraisal Policy for Evidence.
 There is, however, always a race condition possible in that the
 state of the Attester, and the Appraisal Policy for Evidence,
-might change immediately after the Evidence or Attestation Result was generated.
-The goal is merely to narrow the time window to something the Verifier
+might change immediately after the Evidence or Attestation Results were generated.
+Freshness is a key component for enabling caching
+and reuse of both Evidence and Attestation Results, which is especially
+valuable in cases where their computation uses a substantial part of a
+resource budget (e.g., an energy budget in constrained devices).
+The goal is to narrow the time window to something the Verifier
 (for Evidence) or Relying Party (for an Attestation Result) is willing to accept.
 
-There are two common approaches to providing some assurance of freshness.
-The first approach is that a nonce is generated by a remote entity
+There are four common approaches to providing some assurance of freshness, effectively determining the epoch of the
+validity-span of, e.g., Evidence or Attestation Results.
```

Comment:
"epoch of the validity-span" is hard to parse and understand.  Can this be rephrased in layman's terms?

Reply:
This text doesn't exist anymore.